api/swagger.yaml

```diff
@@ -69,14 +80,56 @@ info:
 
     The `serveraddress` is a domain/IP without a protocol. Throughout this structure, double quotes are required.
 
-    If you have already got an identity token from the [`/auth` endpoint](#operation/checkAuthentication), you can just pass this instead of credentials:
+    If you have already got an identity token from the [`/auth` endpoint](#operation/SystemAuth), you can just pass this instead of credentials:
 
     ```
     {
       "identitytoken": "9cbaf023786cd7..."
     }
     ```
 
+# The tags on paths define the menu sections in the ReDoc documentation, so
+# the usage of tags must make sense for that:
+# - They should be plural, not singular.
+# - There should not be too many tags, or the menu becomes unwieldly. For
+#   example, it is preferable to add a path to the "Misc" tag instead of
```

Comment:
Yeah, I considered that, but opted against it because it better categorised the things inside it, and encouraged people to put things inside there instead of a new tag (for the sake of a neater menu). The latter problem is documented now though, so I'll change it.